x-pack/plugins/observability_solution/inventory/public/components/entity_icon/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { AGENT_NAME, CLOUD_PROVIDER } from '@kbn/observability-shared-plugin/common';
+import { type CloudProvider, CloudProviderIcon, AgentIcon } from '@kbn/custom-icons';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import type { AgentName } from '@kbn/elastic-agent-utils';
+import { ENTITY_TYPE } from '../../../common/es_fields/entities';
+import type { Entity } from '../../../common/entities';
+
+interface EntityIconProps {
+  entity: Entity;
+}
+
+export function EntityIcon({ entity }: EntityIconProps) {
+  const entityType = entity[ENTITY_TYPE];
+  const emptyIcon = (
+    <div
+      style={{
+        width: '24px',
+        height: '24px',
+      }}
+    />
+  );
+
+  if (entityType === 'host' || entityType === 'container') {
+    const cloudProvider = entity[CLOUD_PROVIDER] as CloudProvider;
+
+    const formattedCloudProvider =
+      typeof cloudProvider === 'string' ? cloudProvider : cloudProvider?.[0];
+
+    return (
+      <EuiFlexGroup
+        style={{
+          width: '24px',
+          height: '24px',
+        }}
+        alignItems="center"
+        justifyContent="center"
+      >
+        <EuiFlexItem grow={false}>
+          <CloudProviderIcon
+            cloudProvider={formattedCloudProvider}
+            size="m"
```

Comment:
This icon uses `size=m` while agent uses `size=l`.  Can you please make it consistent 🙏

Reply:
You can read [here](https://github.com/elastic/kibana/pull/194543#discussion_r1784030642) the reasoning behind that decision.

Reply:
Oh I see, thanks